data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,305 @@
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage>
+      <start_multipath config:type="boolean">false</start_multipath>
+    </storage>
+  </general>
+  <kdump>
+    <add_crash_kernel config:type="boolean">true</add_crash_kernel>
+    <crash_kernel>201M,high</crash_kernel>
+    <crash_xen_kernel>201M\&lt;4G</crash_xen_kernel>
+  </kdump>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+<interfaces config:type="list">
+  <interface>
+    <device>br0</device>
+    <bootproto>dhcp</bootproto>
+    <bridge>yes</bridge>
+    <bridge_forwarddelay>0</bridge_forwarddelay>
+    <bridge_ports>eth0 eth1</bridge_ports>
+    <bridge_stp>off</bridge_stp>
+    <startmode>auto</startmode>
+  </interface>
+  <interface>
+    <device>eth0</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+  <interface>
+    <device>eth1</device>
+    <bootproto>none</bootproto>
+    <startmode>hotplug</startmode>
+  </interface>
+</interfaces>
+  </networking>
+  <ntp-client>
+    <ntp_policy>auto</ntp_policy>
+    <ntp_servers config:type="list">
+      <ntp_server>
+        <address>europe.pool.ntp.org</address>
+        <iburst config:type="boolean">true</iburst>
+        <offline config:type="boolean">false</offline>
+      </ntp_server>
+    </ntp_servers>
+    <ntp_sync>systemd</ntp_sync>
+  </ntp-client>
+  <partitioning config:type="list">
+    <drive>
+      <!--device>/dev/sda</device-->
+      <disklabel>msdos</disklabel>
+      <enable_snapshots config:type="boolean">false</enable_snapshots>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <create config:type="boolean">true</create>
+          <create_subvolumes config:type="boolean">true</create_subvolumes>
+          <filesystem config:type="symbol">btrfs</filesystem>
+          <format config:type="boolean">true</format>
+          <mount>/</mount>
+          <mountby config:type="symbol">uuid</mountby>
+          <partition_id config:type="integer">131</partition_id>
+          <partition_nr config:type="integer">1</partition_nr>
+          <resize config:type="boolean">false</resize>
+          <size>400G</size>
+        </partition>
+        <partition>
+          <create config:type="boolean">true</create>
+          <filesystem config:type="symbol">swap</filesystem>
+          <format config:type="boolean">true</format>
+          <mount>swap</mount>
+          <mountby config:type="symbol">uuid</mountby>
+          <partition_id config:type="integer">130</partition_id>
+          <partition_nr config:type="integer">2</partition_nr>
+          <resize config:type="boolean">false</resize>
+          <size>20G</size>
```

Comment:
Will it be better to make swap size fitting 2^n? eg 16G or 32G?

Reply:
That's work for small physical memory. 2^n is too big space for large memory. We give 20G to swap in case system is out of memory which I believe, never going to happen as our machines memory are all big.

Reply:
Well, some have 16G which is not that big IMHO. Anyway, we set swap just as any user would do. And for best practice, usually the size is better to be the exponential number of 2, like 2G, 4G,8G, etc. If you choose 20G, better to slightly adjust it to be 16G.

Reply:
yes, 16G is ok.